charts/tfy-agent/values.yaml

```diff
@@ -432,6 +432,70 @@ tfyAgentProxy:
     ## 
     name: ""
 
+  clusterRole:
+    ## @param tfyAgentProxy.clusterRole.enable Create cluster role.
+    enable: true
+    ## @param tfyAgentProxy.clusterRole.rules [array] Cluster role Authz rules.
+    rules:
+      - apiGroups: [""]
+        resources: ["namespaces", "resourcequotas"]
+        verbs: ["get", "delete", "patch", "list", "create"]
+
+      - apiGroups: [""]
+        resources: ["resourcequotas/status"]
+        verbs: ["get"]
+
+      - apiGroups: [""]
+        resources: ["secrets"]
+        verbs: ["create", "patch"]
+
+      - apiGroups: [""]
+        resources: ["pods"]
+        verbs: ["delete"]
+
+      - apiGroups: [""]
+        resources: ["pods/log", "events", "nodes", "services", "persistentvolumes"]
```

Comment:
do we need v1/apps Deployment somewhere?

Reply:
no